modules/@apostrophecms/area/ui/apos/components/AposAreaMenu.vue

```diff
@@ -106,10 +105,6 @@ export default {
     widgetOptions: {
       type: Object,
       required: true
-    },
-    docId: {
```

Comment:
This stuff was unused and was causing an error in the doc editor since there was no doc _id yet.